config.test.yaml

```diff
@@ -159,6 +163,83 @@ templates:
                   options:
                     host: http://appservice.wmflabs.org
 
+      /{module:summary}:
```

Comment:
Ah, this is for no-cache requests. In that case, the back-end module should handle both and the public endpoint should call it. This portion and the public one share a lot of logic, so copy/pasting seems like a bad idea. Also, this should not go into the config, but live in its own specialised module.